docs/fundamentals/tools-and-productivity.md

```diff
@@ -21,7 +21,7 @@ The .NET CLI is a cross-platform toolchain for developing, building, running, an
 
 ## IDEs
 
-You can write .NET applications in [Visual Studio Code](https://code.visualstudio.com/docs), [Visual Studio](/visualstudio/windows), or [Visual Studio for Mac](/visualstudio/mac).
+You can write .NET applications in [Visual Studio Code](https://code.visualstudio.com/docs), [Visual Studio](/visualstudio/windows), [Visual Studio for Mac](/visualstudio/mac), or [JetBrains Rider](https://www.jetbrains.com/rider/).
```

Comment:
```suggestion
You can write .NET applications in [Visual Studio Code](https://code.visualstudio.com/docs), [Visual Studio](/visualstudio/windows), [Visual Studio for Mac](/visualstudio/mac), or with [other .NET tools](https://dotnet.microsoft.com/platform/tools).
```

Since we do link to JetBrains on our marketing page, we could link to that page from here.